ext/phar/tests/bug77022.phpt

```diff
@@ -19,6 +19,8 @@ foreach([Phar::TAR => 'tar', Phar::ZIP => 'zip'] as $mode => $ext) {
 	$phar->addFile($sFile, 'test-file-phar');
 	$phar->addFromString("test-from-string", 'test-file-phar');
 	$phar->extractTo(__DIR__);
+	clearstatcache(__DIR__ . '/test-file-phar');
+	clearstatcache(__DIR__ . '/test-from-string');
```

Comment:
that wouldn't work as you intended, because filename is a second parameter for clearstatcache(), but I've already fixed it.